deepchem/models/IRV.py

```diff
@@ -28,33 +42,38 @@ def __init__(self, n_tasks, K, penalty, **kwargs):
         super(IRVLayer, self).__init__(**kwargs)
 
     def build(self, input_shape):
```

Comment:
Add type annotations.

Also this shouldn't be build (not nn.Module torch convention)